linear_operator/operators/mul_linear_operator.py

```diff
@@ -62,7 +65,7 @@ def _matmul(self, rhs):
             left_res = left_res.view(*output_batch_shape, n, rank, m)
             res = left_res.mul_(left_root.unsqueeze(-1)).sum(-2)
         # This is the case where we're not doing a root decomposition, because the matrix is too small
-        else:
+        else:  # Dead?
```

Comment:
This is b/c the `MulLinearOperator` always constructs a root decomposition?

Reply:
Yes, exactly.